backend/database/schema.sql

```diff
@@ -26,28 +26,28 @@ CREATE TABLE
         role_name VARCHAR(255) NOT NULL
     );
 
-CREATE tABLE
+CREATE TABLE
     user (
         id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
         username VARCHAR(80) NOT NULL,
         lastname VARCHAR(80) NULL,
         firstname VARCHAR(80) NULL,
         email VARCHAR(255) NOT NULL,
         password VARCHAR(255) NOT NULL,
-        score INT NOT NULL DEFAULT 0,
-        created_at TIMESTAMP NOT NULL,
+        score INT NULL DEFAULT 0,
+        created_at DATETIME NULL DEFAULT NOW(),
         city VARCHAR(80) NULL,
         postal_code VARCHAR(255) NULL,
-        role_id INT NOT NULL,
+        role_id INT NULL DEFAULT 2,
         CONSTRAINT fk_user_role FOREIGN KEY(role_id) REFERENCES role(id)
     );
 
 CREATE TABLE
     artwork (
         id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
-        path_pic VARCHAR(255) NOT NULL,
+        path_pic VARCHAR(255) NULL,
         title VARCHAR(255) NULL,
-        adress VARCHAR(255) NOT NULL,
+        coordinates POINT NOT NULL,
```

Comment:
Latitude, longitude fields as we said earlier

Reply:
done!